src/meson.build

```diff
@@ -120,6 +121,7 @@ install_headers('libnvme.h', install_mode: mode)
 install_headers('libnvme-mi.h', install_mode: mode)
 install_headers([
         'nvme/api-types.h',
+        'nvme/nbft.h',
```

Comment:
This is also supposed to be alphabetical sorted.

Reply:
fixed with my latest update.